crates/oxc_allocator/src/vec.rs

```diff
@@ -261,8 +257,7 @@ impl<T: Hash> Hash for Vec<'_, T> {
 
 impl<T: Debug> Debug for Vec<'_, T> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let inner = &*self.0;
-        f.debug_tuple("Vec").field(inner).finish()
+        f.debug_tuple("Vec").field(&self.0).finish()
```

Comment:
Maybe we can simplify this method to just `self.0.fmt(f)`?

I'm not sure. Maybe that'd produce different output.

Reply:
Yes, I apply your suggestion, it is okay to produce different output.

Reply:
Ah. But is it OK? Don't we rely on output being `Vec([...])` in some places (snapshots maybe)?

If it changes the output, let's go back to what you had before. If we want to change it, we can do that in a separate PR.

Reply:
Only a debug test fails, I am ok to do it in a separate PR.

Reply:
Sorry my suggestion was a waste of time!